audit/projects/k8s-infra-prow-build-trusted/secrets/snyk-token/versions.json

```diff
@@ -1,4 +1,76 @@
 [
+  {
+    "createTime": "2021-06-17T17:06:59.041337Z",
+    "etag": "\"15c4f93c2d1439\"",
+    "name": "projects/180382678033/secrets/snyk-token/versions/12",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
+  {
+    "createTime": "2021-06-17T15:23:05.481298Z",
+    "etag": "\"15c4f7c8a08052\"",
+    "name": "projects/180382678033/secrets/snyk-token/versions/11",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
+  {
+    "createTime": "2021-06-17T12:44:28.595177Z",
+    "etag": "\"15c4f5916057e9\"",
+    "name": "projects/180382678033/secrets/snyk-token/versions/10",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
+  {
+    "createTime": "2021-06-17T05:47:19.716124Z",
+    "etag": "\"15c4efbd89f51c\"",
+    "name": "projects/180382678033/secrets/snyk-token/versions/9",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
```

Comment:
@spiffxp is it a good thing to have so many versions for the same secret ? IMHO, we should disable the unused versions.

Reply:
Opened https://github.com/kubernetes/k8s.io/issues/2277 for followup